devicon.json

```diff
@@ -227,6 +227,24 @@
         "color": "#002A41",
         "aliases": []
     },
+    {
```

Comment:
It seems that there are two entries for `bash` in the `devicon.json`. Can you please remove one of them please?